.gitignore

```diff
@@ -0,0 +1,2 @@
+*.swp
+vendor
```

Comment:
Why would you want git to track that? this means that you're going to be uploading your vendor folder to github?? I see your issue though, I would like a nice fast way to deal with the problem of needing to mod vendor packages on the fly without needing to do a glide update. I've sometimes found it useful to pull specific files from a vendor repo to the main repo - This could be a nice mini tool to create in this repo! 
But anyways as I understand it, everything that is tracked should be uploaded. + It's nice to be able to use `git add -A`